pkg/commands/plan/plan.go

```diff
@@ -322,6 +325,30 @@ func (c *PlanCommand) terraformPlan(ctx context.Context) (*RunResult, error) {
 	stdout.Reset()
 	stderr.Reset()
 
+	if !c.skipJSONPlanFile {
```

Comment:
Will this be behind a flag? What is this for since the user cant change this?

Reply:
it's moreso for skipping the json file in unit tests, because it creates a file for every time `go test` is run. I'm trying to see if there's a better way to override the file location to `t.TempDir()` instead of `c.childDir`

Reply:
ah yes, that is tough since were using testdata, I was thinking we should write the out files to a temp dir anyways....so might be time to factor that in.

Reply:
Added a new flag `output-dir` where the binary and json file will be written to. This can be either a child directory or not, but the storage client will continue to use the existing `directory` path.